File: TaxiApp/Common/Models/Ride.cs
namespace Common.Models
{
    public class Ride
    {
        public string StartAddress { get; set; }
        public string Destination { get; set; }
        public decimal Price { get; set; }
        public int WaitTime { get; set; }
        public string Status { get; set; }
    }
}

File: React/empty-react-project-master/src/Models/Ride.js

class RideModel {
    constructor(startAddress, destination, price, waitTime, status){
      this.startAddress = startAddress;
      this.destination = destination;
      this.price = price;
      this.waitTime=waitTime;
      this.status = status;
    }
  }
  
  export default RideModel;